lib/ClimaLandSimulations/src/utilities/pfts.jl

```diff
@@ -0,0 +1,614 @@
+        defined_param_names = keys(parameters)
+        for param in param_list
+            if param ∉ defined_param_names
+                error("PFT $name is missing parameter $param")
+            end
+        end
+        new(name, parameters)
+    end
+end
+
+
+# The default PFTs used in the model use parameter values drawn from defferent 
+# sources in the literature. The sources are listed here with abbreviations that
+# are used in the comments in the code below.
+#
+#    Chen, J. M., G. Mo, J. Pisek, J. Liu, F. Deng, M. Ishizawa, 
+#    and D. Chan (2012), Effects of foliage clumping on the estimation of global
+#    terrestrial gross primary productivity, Global Biogeochem. Cycles, 26, 
+#    GB1019, doi:10.1029/2010GB003996.
+#    abb. as (Chen et al. 2012)
+#
+#    CLM5.0 Technical Note
+#    abb. as (CLM5.0)
+#
+#    Best, M. J., Pryor, M., Clark, D. B., Rooney, G. G., Essery, R. L. H., 
+#    Ménard, C. B., Edwards, J. M., Hendry, M. A., Porson, A., Gedney, N.,
+#    Mercado, L. M., Sitch, S., Blyth, E., Boucher, O., Cox, P. M., Grimmond, 
+#    C. S. B., and Harding, R. J.: The Joint UK Land Environment Simulator 
+#    (JULES), model description – Part 1: Energy and water fluxes, Geosci. Model
+#    Dev., 4, 677–699, https://doi.org/10.5194/gmd-4-677-2011, 2011.
+#    abb as (JULES Model Description) 
+#
+#    Oliver, Rebecca & Mercado, Lina & Clark, Doug & Huntingford,
+#    Chris & Taylor, Christopher & Vidale, P.L. & McGuire, Patrick & Todt, 
+#    Markus & Folwell, Sonja & Semeena, Valiyaveetil & Medlyn, Belinda. (2022).
+#    Improved representation of plant physiology in the JULES-vn5.6 land surface
+#    model: Photosynthesis, stomatal conductance and thermal acclimation. 
+#    10.5194/gmd-2022-11. 
+#    abb. as (Oliver et al. 2022)
+
+# Define the default PFTs that are used in the model.
+
+NET_Temp = Pft(
+    "Temperate Needleleaf Evergreen Tree",
+    (
+        # Radiative Transfer Model
+        Ω = 0.74,          # Chen et al. 2012 Table 1
+        α_PAR_leaf = 0.07, # CLM5.0 Table 2.3.1
+        α_NIR_leaf = 0.35, # CLM5.0 Table 2.3.1
+        τ_PAR_leaf = 0.05, # CLM5.0 Table 2.3.1
+        τ_NIR_leaf = 0.10, # CLM5.0 Table 2.3.1
+
+        # Canopy Energy Model
+        ac_canopy = 2.5e3, # Taken from site params
+
+        # Stomatal Conductance Model
+        g1 = 2.35, # CLM5.0 Table 2.9.1
+
+        # Photosynthesis Model,
+        Vcmax25 = 5.080e-5, # Oliver et al. 2022 Table 2
+
+        # Hydraulics Model and Plant Parameters
+        SAI = 2.50,                # Guess
+        f_root_to_shoot = 3.50,    # Guess
+        K_sat_plant = 2.78e-6,     # Guess
+        ψ63 = -4.0 / 0.0098,       # Guess
+        Weibull_param = 4.00,      # Taken from site params
+        a = 0.05 * 0.0098,         # Taken from site params
+        capacity = 15.00,          # Guess
+        plant_S_s = 1e-2 * 0.0098, # Taken from site params
+        rooting_depth = 3.90,      # Bonan Table 2.3
```

Comment:
Any help finding sources for plant hydraulics parameters or accurate guesses of the values would be really helpful. I haven't found anywhere in the literature matching these values to PFTs. 

Similarly for the `ac_canopy`.

Reply:
We can also get rooting depth from this beta parameter: CLM Table 2.11.1, in the chapter on plant hydraulics. I can look this up tomorrow (a little math to convert from beta to rooting depth) if Bonan doesnt have it

plant_S_s can be constant. It applies in a regime that Im not sure is super physical in plants - when the plant pores are saturated. im not sure how important it is but I dont think we have data on it.

CLM says that the root to shoot ratio is a PFT parameter...root-to-shoot ratio (PFT parameter) 
and I think a lot of these plant hydraulics params should be too. Maybe @braghiere knows if there is a file with them if they are not published in the docs?

Reply:
The acronym for this parameter is froot_leaf    = allocation parameter: new fine root C per new leaf C (gC/gC). Refer to: https://escomp.github.io/ctsm-docs/versions/release-clm5.0/html/tech_note/Plant_Hydraulics/CLM50_Tech_Note_Plant_Hydraulics.html#vertical-root-distribution

See CLM PFT file example: https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/pftdata/pft-physiology-cn.c070601.readme